lib/firmata.js

```diff
@@ -545,6 +545,17 @@ var Board = function(port, options, callback) {
         this.setSamplingInterval(settings.samplingInterval);
       }
       if (settings.skipCapabilities) {
+        this.analogPins = this.settings.analogPins || [ 14, 15, 16, 17, 18, 19 ];
```

Comment:
or skip setting analog pin if they don't specify any. maybe the user doesn't intend to use any analog pins